packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -214,7 +219,8 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
         // Sanity check that pool has enough balance to cover relay amount + proposer reward. Reward amount will be
         // paid on settlement after the OptimisticOracle price request has passed the challenge period.
         require(
-            l1Token.balanceOf(address(this)) >= amount + _getProposerRewardAmount(proposerRewardPct, amount),
+            l1Token.balanceOf(address(this)) >=
+                amount + _getAmountFromPct(slowRelayFeePct + instantRelayFeePct, amount),
```

Comment:
I thought amount already included the relay fees? How is amount computed by the relayer based on the deposit size?

Reply:
o that is true. the original require statement was more correct. will revert that.